gapic/schema/api.py

```diff
@@ -670,6 +678,63 @@ def enforce_valid_method_settings(
         if all_errors:
             raise MethodSettingsError(yaml.dump(all_errors))
 
+    @cached_property
+    def all_library_settings(
+        self,
+    ) -> Mapping[str, Sequence[client_pb2.ClientLibrarySettings]]:
+        """Return a map of all `google.api.client.ClientLibrarySettings` to be used
+        when generating client libraries.
+        https://github.com/googleapis/googleapis/blob/master/google/api/client.proto#L130
+
+        Return:
```

Comment:
nit: `Returns`.
Same for the other function.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2098/commits/e0d252e8cc474efd199bd6d8d1d44ec2806d997c